src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import users from '../../api/users';
+import { ToDos } from '../../react-app-env';
+import TodoInfo from '../TodoInfo/TodoInfo';
+import UserInfo from '../UserInfo/UserInfo';
+import './TodoList.css';
+
+interface Props {
+  initialTodos: ToDos[];
+}
+
+const TodoList: React.FC<Props> = ({ initialTodos }) => {
+  const [todos, setTodos] = useState(initialTodos);
+  const [newTitle, setNewTitle] = useState('');
+  const [selectedUser, setSelectedUSer] = useState(0);
+  const [showTitleError, setShowTitleError] = useState(false);
+  const [showSelectErrorMessage, setShowSelectErrorMessage] = useState(false);
+
+  function addTodo() {
+    const newTodo = {
+      title: newTitle,
+      userId: selectedUser,
+      completed: false,
+      id: todos.length + 1,
```

Comment:
> - `id` is a maximal `id` in the array + 1

not `list.length`
something like this:
`Math.max(...array.map((todo) => todo.id)) + 1`